perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -1589,7 +1591,8 @@ sub waste_munge_problem_form_fields {
         },
         497 => {
             container_name => "General",
-            label => "The bin wasn’t returned to the collection point",
+            label => "The bin wasn’t returned to the collection point (Please phone 01733 747474 to report this issue)",
```

Comment:
Could this label change depending on `$not_staff`? Think you'd be able to drop the `report_new_munge_before_insert` then too maybe?